src/Commands/ToggleDone.ts

```diff
@@ -34,15 +34,13 @@ export const toggleDone = (checking: boolean, editor: Editor, view: View) => {
     const toggledLine = toggleLine({ line, path });
     editor.setLine(lineNumber, toggledLine);
 
-    // The cursor is moved to the end of the line by default.
-    // If there is text on the line, put the cursor back where it was on the line.
-    if (/[^ [\]*-]/.test(toggledLine)) {
-        editor.setCursor({
-            line: cursorPosition.line,
-            // Need to move the cursor by the distance we added to the beginning.
-            ch: cursorPosition.ch + toggledLine.length - line.length,
-        });
-    }
+    // The cursor is moved to either the beginning of the line (usually) or the end of the line (if it was already at the end) by default.
+    // Put the cursor back where it was on the line.
+    editor.setCursor({
+        line: cursorPosition.line,
+        // Need to move the cursor by the distance we added to the beginning.
+        ch: cursorPosition.ch + toggledLine.length - line.length,
+    });
```

Comment:
I have got stuck several times reviewing this section in the Web UI.

I cannot work out why the `if` condition is no longer required.

What am I missing?